src/main/java/org/randywebb/wlts/beans/Teacher.java

```diff
@@ -9,7 +9,7 @@
 //import org.slf4j.Logger;
 //import org.slf4j.LoggerFactory;
 
-/** Represents a minister (previously home or visiting teacher) */
+/** Represents a minister (previously home or visiting teacher). */
```

Comment:
Hmm, is it worth refactoring this class to call it "Minister" now? probably not, but it wouldn't be too difficult to do.

Reply:
I'm questioning if this API will break down entirely in August anyway. The API still calls it HT and VT

Reply:
Good point. Let's not refactor now, but watch and see what happens to their API. We can adjust this code when we need to update for their likely API change.